packages/appkit/src/client.ts

```diff
@@ -2093,4 +2099,68 @@ export class AppKit {
 
     return this.initPromise
   }
+
+  private async checkExistingConnection() {
```

Comment:
before doing any of this, shouldn't you check if you are using the corresponding flags? ie features.social

Reply:
we check if the required login uri is present in the URL already, flow wouldn't continue if its missing

Reply:
what happens if the login url is there but the config does not allow for socials ?=

Reply:
I think it'd be better to call this from `createAuthProvider` or `syncAuthProvider` where you have certainty this is potentially needed